otx/cli/tools/build.py

```diff
@@ -19,17 +19,29 @@
 
 import argparse
 import os
+from pathlib import Path
 
 from otx.cli.builder import Builder
 from otx.cli.utils.importing import get_otx_root_path
+from otx.core.config.manager import ConfigManager
+from otx.core.data.manager.dataset_manager import DatasetManager
 
 SUPPORTED_TASKS = ("CLASSIFICATION", "DETECTION", "INSTANCE_SEGMENTATION", "SEGMENTATION")
 SUPPORTED_TRAIN_TYPE = ("incremental", "semisl", "selfsl")
 
+def set_workspace(path, task, model):
```

Comment:
This function return path if path is not None. then, how about removing path as argument and call this function `if path is None`?